features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageWireframeHelper.kt

```diff
@@ -129,6 +128,23 @@ internal class ImageWireframeHelper(
         return result
     }
 
+    private fun resolveDrawableProperties(view: View, drawable: Drawable?): DrawableProperties {
+        if (drawable == null) return DrawableProperties(null, 0, 0)
+
+        return when (drawable) {
+            is LayerDrawable -> {
+                if (drawable.numberOfLayers > 0) {
+                    resolveDrawableProperties(view, drawable.getDrawable(0))
+                } else {
+                    DrawableProperties(drawable, drawable.intrinsicWidth, drawable.intrinsicHeight)
```

Comment:
Perhaps the drawable should be returned here as null

Reply:
so basically we do not care anymore about `RippleDrawable` but about his super class. Much more reliable this way I'd say.